api/.eslintrc

```diff
@@ -21,6 +49,14 @@
         "no-console": "off",
         "compat/compat": "error"
       }
+    },
+    {
+      "files": [
+        "tests/**/*.js"
+      ],
+      "rules": {
+        "node/no-extraneous-require": "off"
```

Comment:
Yes, correct. It complained about sinon and chai being being extraneous. Instead of maintaining one enormous lists for tests alone, I just turned it off.